tools/clang/test/HLSLFileCheck/passes/dxil/dxil_remove_dead_pass/dxil_ops.hlsl

```diff
@@ -0,0 +1,38 @@
+// RUN: %dxc %s -T ps_6_0 -Od | FileCheck %s
+
+// CHECK: @main
+
```

Comment:
I'm not sure what this is testing for.

Reply:
I think just that it successfully resolves the resource being used by simplifying the mads, and therefore successfully compiles the shader.  But a comment wouldn't hurt.

Reply:
That's right. Added a comment to explain.